clients/common/nm-secret-agent-simple.c

```diff
@@ -1151,8 +1151,10 @@ nm_secret_agent_simple_response (NMSecretAgentSimple *self,
 
 		g_variant_builder_init (&conn_builder, NM_VARIANT_TYPE_CONNECTION);
 		g_hash_table_iter_init (&iter, settings);
-		while (g_hash_table_iter_next (&iter, (gpointer *) &name, (gpointer *) &setting_builder))
+		while (g_hash_table_iter_next (&iter, (gpointer *) &name, (gpointer *) &setting_builder)) {
 			g_variant_builder_add (&conn_builder, "{sa{sv}}", name, setting_builder);
+			g_variant_builder_unref (setting_builder);
```

Comment:
I think it would be nicer if the hash table frees it with a GDestroyNotify function

Reply:
Done